### PR TITLE
init beyond theme

### PR DESCRIPTION
**Pull request by aprilyang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "author": "zhouyang@yunify.com",
```
Comment (Xuanwo) [inline package.json]: `author` should be beyondstorage

File: package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "license": "MIT",
```
Comment (Xuanwo) [inline package.json]: Please use Apache 2.0

**Comment by Xuanwo** [comment]
ping @aprilyang to take a look.

File: package.json (hunk 1 above)
Comment (aprilyang) [reply package.json]: Fixed

File: package.json (hunk 2 above)
Comment (aprilyang) [reply package.json]: Fixed

**Pull request merged**